python/lsst/sconsUtils/state.py

```diff
@@ -258,7 +258,7 @@ def ClassifyCc(context):
     else:
         if env['cc'] != '':
             CC = CXX = None
-            if re.search(r"^gcc(-\d+(\.\d+)*)?( |$)", env['cc']):
+            if re.search(r"^gcc-(\d+(\.\d+)*)?( |$)", env['cc']):
```

Comment:
so this regexp no longer matches `gcc` ? Maybe I'm missing some earlier context.